src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -306,6 +306,54 @@ static void add_pending_ping(pending_ping** root, grpc_closure* on_initiate,
  */
 typedef struct rr_connectivity_data rr_connectivity_data;
 
+typedef struct glb_lb_call_data {
```

Comment:
just a suggestion: you may want to take this opportunity to turn this into a C++ class, inheriting from [RefCountedWithTracing](https://github.com/grpc/grpc/blob/master/src/core/lib/support/ref_counted.h#L67). That's what I've done in my subchannel refactoring PR (see [here](https://github.com/grpc/grpc/pull/13932/files#diff-4ae77ab90fbeb7fc0f5dba398b2b63afR76))

Reply:
I see. I will change that. Thanks!